mlir/include/mlir/Dialect/Linalg/IR/LinalgStructuredOps.td

```diff
@@ -551,6 +551,136 @@ def BroadcastOp : LinalgStructuredBase_Op<"broadcast", [
   let hasCanonicalizer = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// Op definition for ElemwiseOp - with user-defined maps, computation type etc.
+//===----------------------------------------------------------------------===//
+
+def ElemwiseOp : LinalgStructuredBase_Op<"elemwise", [
```

Comment:
Nit: I'd rather just use `ElementWiseOp` and `element_wise` . But totally my preference.

Reply:
+1 for full spelling, feels more intuitive
nit to nit: I'd go with a single word version `ElementwiseOp` and `elementwise`, it's also more consistent with spelling across repo